.github/ISSUE_TEMPLATE/03_mybuildisbroken.md

```diff
@@ -10,9 +10,9 @@ labels: needs-triage
 
 <!--
 * Here are some useful links to help you figure out what's wrong.
-* Our wiki: https://github.com/microsoft/msbuild/blob/master/documentation/wiki/Home.md
-* General Help: https://github.com/microsoft/msbuild/blob/master/documentation/wiki/Something's-wrong-in-my-build.md
-* Tips & Tricks: https://github.com/microsoft/msbuild/blob/master/documentation/wiki/MSBuild-Tips-%26-Tricks.md
+* Our documentation: https://github.com/dotnet/msbuild/blob/main/documentation/README.md
+* General Help: https://github.com/microsoft/msbuild/blob/main/documentation/wiki/Something's-wrong-in-my-build.md
+* Tips & Tricks: https://github.com/microsoft/msbuild/blob/main/documentation/wiki/MSBuild-Tips-%26-Tricks.md
```

Comment:
```suggestion
* General Help: https://github.com/dotnet/msbuild/blob/main/documentation/wiki/Something's-wrong-in-my-build.md
* Tips & Tricks: https://github.com/dotnet/msbuild/blob/main/documentation/wiki/MSBuild-Tips-%26-Tricks.md
```

Reply:
Thank you @vlada-shubina &mdash; I have pushed this in [commit `00bdb0046c`](https://github.com/dotnet/msbuild/pull/8437/commits/00bdb0046c95dfacd8722ab3f306260feeefc20e), along with reverting the changes to `eng/common`.